docs/pages/setup/admin/troubleshooting.mdx

```diff
@@ -2,22 +2,27 @@
 title: Troubleshooting
 description: Troubleshooting and Collecting Metrics of Teleport Processes
 ---
+<Notice scope={["cloud"]} type="tip">
+
+These instructions apply to `teleport` processes running on Teleport Nodes. In
+Teleport Cloud, the Auth and Proxy Services are monitored and managed for you.
+
+</Notice>
 
 ## Troubleshooting
 
-To diagnose problems you can configure [`teleport`](../reference/cli.mdx#teleport) to
-run with verbose logging enabled by passing it `-d` flag.
+To diagnose problems you can configure the `teleport` process to run with
+verbose logging enabled by passing it the `-d` flag.
 
-<Admonition
-  type="tip"
-  title="NOTE"
+<Notice
+  type="warning"
 >
   It is not recommended to run Teleport in production with verbose logging as it generates a substantial amount of data.
-</Admonition>
+</Notice>
 
-Sometimes you may want to reset [`teleport`](../reference/cli.mdx#teleport) to a clean
-state. This can be accomplished by erasing everything under `"data_dir"`
-directory. Assuming the default location, `rm -rf /var/lib/teleport/*` will do.
+Sometimes you may want to reset the `teleport` process to a clean state. This
+can be accomplished by erasing everything under the `"data_dir"` directory.
+Assuming the default location, `rm -rf /var/lib/teleport/*` will do.
```

Comment:
Maybe make this `rm -rfi /var/lib/teleport/*` just in case someone is typing this manually and absentmindedly types `rm -rf /var/lib` by mistake.